allauth/account/forms.py

```diff
@@ -127,8 +134,9 @@ def __init__(self, *args, **kwargs):
             assert app_settings.AUTHENTICATION_METHOD \
                 == AuthenticationMethod.USERNAME_EMAIL
             login_widget = forms.TextInput(attrs={'placeholder':
-                                                  _('Username or e-mail'),
-                                                  'autofocus': 'autofocus'})
+                                                  _('Username e-mail'),
+                                                  'autofocus': 'autofocus',
+                                                  'autocomplete': 'username'})
```

Comment:
Wouldn't autocomplete=login make more sense, given that it is either the username or the email?